packages/manager/apps/pci-object-storage/src/pages/objects/container/object/show/ActionsComponent.tsx

```diff
@@ -31,6 +38,15 @@ export default function ActionsComponent({
 
   const { projectId } = useParams();
 
+  const shouldShowVersions = useMemo(() => {
```

Comment:
Could you have more meaningful name, I don't understand the difference between shouldSeeVersions and shouldShowVersions